Tools/cases_generator/mypy.ini

```diff
@@ -11,3 +11,5 @@ strict = True
 strict_concatenate = True
 enable_error_code = ignore-without-code,redundant-expr,truthy-bool,possibly-undefined
 warn_unreachable = True
+allow_redefinition = True
+implicit_reexport = True
```

Comment:
Suggestion: move these lines of the config file above the `# ...And be strict:` comment on line 9, since they don't relate to increasing the strictness of mypy